data/json/monsters/zed_nomex.json

```diff
@@ -0,0 +1,78 @@
+    "copy-from": "mon_zombie_hunter",
+    "default_faction": "zombie",
+    "color": "brown_yellow",
+    "symbol": "Z",
+    "hp": 120,
+    "armor": { "bash": 15, "cut": 18, "acid": 14, "heat": 22, "bullet": 11, "electric": 4 },
+    "harvest": "zombie_nomex",
+    "upgrades": false,
+    "death_drops": "mon_zombie_fireman_death_drops",
+    "extend": {
+      "weakpoint_sets": [ "wps_humanoid_turnout_gear", "wps_humanoid_full_helmet" ],
+      "families": [ "prof_wp_syn_armored", "prof_wp_nat_armored" ],
+      "flags": [ "ACIDPROOF", "FIREPROOF", "CLIMBS" ]
+    },
+    "delete": { "special_attacks": [ "bite_humanoid" ] }
+  },
+  {
+    "id": "mon_zombie_fireproof_grappler",
+    "type": "MONSTER",
+    "name": { "str": "fireproof grappler" },
+    "description": "this zombie appears to have once been a firefighter or similar first responder, its body having ripped through and absorbed the majority of its uniform.  Its torso and back are swollen, and its muscles are marbled with knotted growths resembling the nomex fibers of its former gear, feeding into lanky arms that drag on the floor.",
+    "copy-from": "mon_zombie_grappler",
+    "default_faction": "zombie",
+    "symbol": "Z",
+    "armor": { "bash": 15, "cut": 18, "acid": 14, "heat": 22, "bullet": 11, "electric": 4 },
+    "hp": 120,
+    "color": "black_yellow",
+    "harvest": "zombie_nomex",
+    "upgrades": false,
+    "death_drops": "mon_zombie_fireman_death_drops",
+    "extend": {
+      "weakpoint_sets": [ "wps_humanoid_turnout_gear", "wps_humanoid_full_helmet" ],
+      "families": [ "prof_wp_syn_armored", "prof_wp_nat_armored" ],
+      "flags": [ "ACIDPROOF", "FIREPROOF", "RANGED_ATTACKER" ]
+    },
+    "delete": { "special_attacks": [ "bite_humanoid" ] }
+  },
+  {
+    "id": "mon_zombie_fireproof_brute",
+    "type": "MONSTER",
+    "name": { "str": "fireproof brute" },
+    "description": "this zombie appears to have once been a firefighter or similar first responder, but the uniform is hardly recognizable at this point, as the thing has grown new flesh and muscle overtop.  Its sinewy body has taken on a composition similar to the uniforms fire-resistant fabric, most of it congealed into a pair of clubs on the ends of its hands.",
+    "copy-from": "mon_zombie_brute",
+    "default_faction": "zombie",
+    "symbol": "Z",
+    "armor": { "bash": 15, "cut": 18, "acid": 14, "heat": 22, "bullet": 11, "electric": 4 },
+    "color": "red_yellow",
+    "harvest": "zombie_nomex",
+    "upgrades": false,
+    "death_drops": "mon_zombie_fireman_death_drops",
+    "hp": 120,
+    "extend": {
+      "weakpoint_sets": [ "wps_humanoid_turnout_gear", "wps_humanoid_full_helmet" ],
```

Comment:
I don't think
`but the uniform is hardly recognizable at this point, as the thing has grown new flesh and muscle overtop`
Would still have these weakpoint sets if it's merged with the uniform fully